pitest-entry/src/test/java/org/pitest/mutationtest/incremental/IncrementalAnalyserTest.java

```diff
@@ -59,7 +60,7 @@ public void shouldStartPreviousSurvivedMutationsAtAStatusOfNotStartedWhenCoverag
     setHistoryForAllMutationsTo(DetectionStatus.SURVIVED);
     when(
         this.history.hasCoverageChanged(any(ClassName.class),
-            any(BigInteger.class))).thenReturn(true);
+            isNull())).thenReturn(true);
```

Comment:
This change doesn't feel right. DefaultCodeHistory.hasCoverageChanged will throw a NullPointerException when called with NULL. I think we should put more effort into mocking in order to test correct interaction with collaborators:
```java
    when(this.coverage.getCoverageIdForClass(ClassName.fromString("clazz")))
            .thenReturn(BigInteger.valueOf(42L));
    when(this.history.hasCoverageChanged(ClassName.fromString("clazz"), BigInteger.valueOf(42L)))
            .thenReturn(true);
```
(feel free to extract constants and beautify the code above)

Reply:
(also welcome back @StefanPenndorf, not seen you for a long while).

Reply:
fixed.